draft-ietf-httpbis-client-hints.md

```diff
@@ -199,22 +199,10 @@ This document defines the "Accept-CH" and "Accept-CH-Lifetime" HTTP response fie
 
 --- back
 
-# Interaction with Key Response Header Field
-
-Client Hints may be combined with Key response header field ({{KEY}}) to enable fine-grained control of the cache key for improved cache efficiency. For example, the server can return the following set of instructions:
-
-~~~ example
-  Key: Sec-CH-Example;partition=1.5:2.5:4.0
-~~~
-
-Above example indicates that the cache key needs to include the value of the Sec-CH-Example header field with three segments: less than 1.5, 1.5 to less than 2.5, and 4.0 or greater.
-
-~~~ example
-  Key: Width;Sec-CH-Example=320
-~~~
-
-Above example indicates that the cache key needs to include the value of the Sec-CH-Example header field and be partitioned into groups of 320: 0-320, 320-640, and so on.
+# Interaction with Variants Response Header Field
 
+Client Hints may be combined with Variants response header field (<https://tools.ietf.org/html/draft-ietf-httpbis-variants-05>) to enable fine-grained control of the cache key for improved cache efficiency.
```

Comment:
Please use `{{I-D.ietf-httpbis-variants}}` rather than a URI.

```suggestion
Client Hints may be combined with Variants response header field {{?VARIANTS=I-D.ietf-httpbis-variants}} to enable fine-grained control of the cache key for improved cache efficiency.
```